documentation/doc.html

```diff
@@ -689,73 +694,117 @@ <h4>List Item 3</h4>
+          <iframe src="https://www.thiscodeworks.com/embed/6204e0b6c6efb50015062e43" style="width: 100%; height: 177px;"
+            frameborder="0"></iframe>
         </div>
       </div>
 
       <!-- Nav-bar -->
       <div class="component-doc-container">
         <h1 id="nav-bar">Navigation Bar</h1>
         <p class="component-information">
-          A navigation bar is a navigation header that is placed at the top of the page. which is used for navigating through the webpage.
+          A navigation bar is a navigation header that is placed at the top of the page. which is used for navigating
+          through the webpage.
         </p>
         <div class="margin-nav">
           <div class="comp-header">
-          <a href="#" class="nav-logo"> <i class="fas fa-shopping-basket"></i> Store </a>
+            <a href="#" class="nav-logo"> <i class="fas fa-shopping-basket"></i> Store </a>
 
-          <div class="navbar-comp">
+            <div class="navbar-comp">
               <a href="#">home</a>
               <a href="#">features</a>
               <a href="#">products</a>
               <a href="#">categories</a>
               <a href="#">review</a>
               <a href="#">blogs</a>
-          </div>
-      
-          <div class="nav-icons">
+            </div>
+
+            <div class="nav-icons">
               <div class="fas fa-search" id="search-btn"></div>
               <div class="fas fa-filter" id="filter-btn"></div>
               <div class="fas fa-shopping-cart" id="cart-btn"></div>
               <div class="fas fa-user" id="login-btn"></div>
+            </div>
           </div>
         </div>
-        </div>
         <div>
-          <iframe src="https://www.thiscodeworks.com/embed/620a3768c055f30015380b7a" style="width: 100%; height: 86px;" frameborder="0"></iframe>        </div>
+          <iframe src="https://www.thiscodeworks.com/embed/620a3768c055f30015380b7a" style="width: 100%; height: 86px;"
+            frameborder="0"></iframe> </div>
       </div>
+      <!-- Modal -->
+      <div class="component-doc-container">
+        <h1 id="modal">Modal</h1>
+        <p class="component-information">
+          The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, or whatever else.
+        </p>
+        <div class="component-view-container flex-column">
+          <button class="btn primary-btn open-model">
+            Click to Open Modal
+          </button>
+          <div class="modal-background increase-size-modal">
+            <div class="modal">
+              <div class="modal-head">
+                <h3>Basic Modal</h3>
+                <h3 id="modal-close">X</h3>
+              </div>
+              <hr class="break-modal" />
+              <div class="modal-body">
+                <h4>Modal Title</h4>
+                <p>modal content...</p>
+                <p>action saggestion...</p>
+                <p>some content...</p>
```

Comment:
i think you can fill in some mock data here if you want.

Reply:
yes i can